gamemodes/terrortown/entities/weapons/weapon_tttbasegrenade.lua

```diff
@@ -305,3 +312,46 @@ function SWEP:OnRemove()
 		RunConsoleCommand("use", "weapon_ttt_unarmed")
 	end
 end
+
+if CLIENT then
+	local draw = draw
+	local TryT = LANG.TryTranslation
+	local hudTextColor = Color(255, 255, 255, 180)
+
+	---
+	-- @ignore
+	function SWEP:DrawHUD()
+		if self.HUDHelp then
+			self:DrawHelp()
+		end
+
+		if self:GetPin() and self:GetPullTime() > 0 then
+			local client = LocalPlayer()
+
+			local x = ScrW() / 2.0
+			local y = ScrH() / 2.0
+			y = y + (y / 3)
+
+			local pct = 1 - math.Clamp((CurTime() - self:GetPullTime()) / (self:GetDetTime() - self:GetPullTime()), 0, 1)
+
+			local scale = appearance.GetGlobalScale()
+			local w, h = 100 * scale, 20 * scale
+			local drawColor = appearance.SelectFocusColor(client:GetRoleColor())
+
+			draw.AdvancedText(
+				TryT("grenade_fuse"),
+				"PureSkinBar",
+				x - w / 2,
+				y - h,
+				hudTextColor,
+				TEXT_ALIGN_LEFT,
+				TEXT_ALIGN_BOTTOM,
+				true,
+				scale
+			)
+			draw.Box(x - w / 2 + scale, y - h + scale, w * pct, h, COLOR_BLACK)
+			draw.OutlinedShadowedBox(x - w / 2, y - h, w, h, scale, drawColor)
+			draw.Box(x - w / 2, y - h, w * pct, h, drawColor)
+		end
+	end
+end
```

Comment:
Please always add an empty line at the file end

Reply:
i usually do, but some things like merges or something will trim it, isn't there a vscode rule to enforce this? same like the one that trims all whitespace at the end of lines

Reply:
Probably there is something, I don't know though